docs/sidebars.js

```diff
@@ -210,6 +210,11 @@ const sidebars = {
           label: 'Troubleshooting',
           id: 'workflows/troubleshooting',
         },
+        {
+          type: 'doc',
+          label: 'Terraform module',
```

Comment:
```suggestion
          label: 'Use the Terraform module',
```
for consistency with the other workflows